src/components/blocks/StaticBlock.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react';
+
+export default class StaticBlock extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      collectorsExpanded: true,
+      comparatorsExpanded: true,
+      urlsExpanded: true
+    }
+
+    this.handleExpanding = this.handleExpanding.bind(this);
+  }
+
+  toggleExpanding(lists, arrow, isExpanded) {
+    lists.forEach((listItem) => {
+      if(isExpanded) {
+        listItem.style.height = 0;
+        arrow.style.transform = "rotate(90deg)";
+      } else {
+        listItem.style.height = "100%";
+        arrow.style.transform = "rotate(0deg)";
+      }
+    });
+  }
+
+  handleExpanding() {
+    if(this.props.expandable) {
+      const category = this.props.value.toString().toLowerCase();
+      if(category === "collectors") {
+        const modifiers = document.querySelector(".modifiers-list");
```

Comment:
If this is part of react app you should avoid querying for elements, using component communication method like redux instead

Reply:
Done.